RxCocoa/iOS/Proxies/RxCollectionViewDataSourcePrefetchingProxy.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  RxCollectionViewDataSourcePrefetchingProxy.swift
+//  RxCocoa
+//
+//  Created by Rowan Livingstone on 2/15/18.
+//  Copyright © 2018 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+
+import UIKit
+import RxSwift
+
+@available(iOS 10.0, tvOS 10.0, *)
+extension UICollectionView: HasPrefetchDataSource {
+    public typealias PrefetchDataSource = UICollectionViewDataSourcePrefetching
+}
+
+@available(iOS 10.0, tvOS 10.0, *)
+let collectionViewPrefetchDataSourceNotSet = CollectionViewPrefetchDataSourceNotSet()
```

Comment:
I think that we can make this `fileprivate`. (and also make the other corresponding fields like `collectionViewDataSourceNotSet` `fileprivate`).